.env.example

```diff
@@ -2,26 +2,26 @@ SECRET_KEY=
 NODE_ENV=
 PORT=
 DATABASE_URL=
-DEV_DATABASE_URL=
-TEST_DATABASE_URL=
+DATABASE_URL_TEST=
 AUTHOSHAVEN_USER=
 AUTHOSHAVEN_PASS=
 BASE_URL=
 
-FCBK_ID=
-FCBK_APP_SECRET=
+FCBK_ID=392129074733761
```

Comment:
We don't put secret key values in `.env.example`

Reply:
Oooh my bad, corrected it